pkg/cmd/render/env.go

```diff
@@ -68,41 +69,9 @@ func getEtcdName(e *envVarData) (map[string]string, error) {
 	}, nil
 }
 
-func getHeartbeatInterval(e *envVarData) (map[string]string, error) {
-	heartbeat := "100"
-
-	switch configv1.PlatformType(e.platform) {
-	case configv1.AzurePlatformType:
-		heartbeat = "500"
-	case configv1.IBMCloudPlatformType:
-		switch configv1.IBMCloudProviderType(e.platformData) {
-		case configv1.IBMCloudProviderTypeVPC:
-			heartbeat = "500"
-		}
-	}
-
-	return map[string]string{
-		"ETCD_HEARTBEAT_INTERVAL": heartbeat,
-	}, nil
-}
-
-func getElectionTimeout(e *envVarData) (map[string]string, error) {
-	timeout := "1000"
-
-	switch configv1.PlatformType(e.platform) {
-	case configv1.AzurePlatformType:
-		timeout = "2500"
-	case configv1.IBMCloudPlatformType:
-		switch configv1.IBMCloudProviderType(e.platformData) {
-		case configv1.IBMCloudProviderTypeVPC:
-			timeout = "2500"
-		}
-	}
-
-	return map[string]string{
-		"ETCD_ELECTION_TIMEOUT": timeout,
-	}, nil
-
+func getTuningProfileValues(e *envVarData) (map[string]string, error) {
+	// TODO (alray) Can the profile be set before the bootstrap is created?
```

Comment:
// TODO Remove me ;)

Reply:
the infra CRD should be available here:

https://github.com/openshift/cluster-etcd-operator/blob/master/pkg/cmd/render/render.go#L260-L264

so shouldn't be hard to directly set this to the right profile. It would safe a static pod rollout, I reckon...